### PR TITLE
feat: HoC for Bridge User and Liquidity Provider

### PR DESCRIPTION
**Pull request by Tulun**
1. HoC for Bridge User
2. HoC for LP Card
3. Fix for UI design

Screen Shot 2022-09-26 at 2 07 11 PM

Screen Shot 2022-09-26 at 2 06 02 PM

### CODE REVIEW

File: src/views/PreLaunchAirdrop/components/CardContent.tsx [hunk 1]
```diff
@@ -102,21 +106,30 @@ const ExternalLink = styled(Link)`
...
+  box-sizing: border-box;
   display: flex;
   flex-direction: row;
   align-items: center;
-  justify-content: flex-start;
+  justify-content: center;
+  padding: 6px 16px;
   gap: 8px;
+  height: 48px;
+  min-width: 135px;
```
[the remark this answers is not in the document]
Comment (Tulun) [reply src/views/PreLaunchAirdrop/components/CardContent.tsx:117]: Doesn't work for some reason. 
Screen Shot 2022-09-26 at 2 18 06 PM

File: src/views/PreLaunchAirdrop/api/getPrelaunchRewards.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ export default async function getPrelaunchRewards(address: string) {
     liquidityProviderRewards: {
       walletEligible: true,
       completed: false,
-      payout: "5",
+      payout: "5.123412341242314",
```
[the remark this answers is not in the document]
Comment (Tulun) [reply src/views/PreLaunchAirdrop/api/getPrelaunchRewards.ts:32]: I thought Alex was going to return this as a normalized string, but I'm happy to change it to whatever.

File: src/views/PreLaunchAirdrop/components/cards/LiquidityProviderCard.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import RewardsCard from "./RewardsCard";
+import { RewardsApiInterface } from "utils/serverless-api/types";
+
+import { ReactComponent as MoneyIcon } from "assets/icons/plaap/money.svg";
+import { ReactComponent as WalletIcon } from "assets/icons/wallet-icon.svg";
+import { CheckIconState } from "../CardIcon";
+
+interface Props {
+  rewardsData: RewardsApiInterface;
+  account: string | undefined;
```
[the remark this answers is not in the document]
Comment (Tulun) [reply src/views/PreLaunchAirdrop/components/cards/LiquidityProviderCard.tsx]: Done!

File: src/views/PreLaunchAirdrop/components/CardContent.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Tulun) [reply src/views/PreLaunchAirdrop/components/CardContent.tsx]: done!

**Review by dohaki — APPROVED** [review]
Cool LGTM. Only the stuff @james-a-morris mentioned regarding the breaking change.

**Pull request merged**